dashboard/dashboard.py

```diff
@@ -0,0 +1,83 @@
+"""Streamlit Dashboard main script."""
+
+import re
+import streamlit as st
+
+
+def is_valid_email(email: str) -> bool:
+    """Validates inputted user email address."""
+    email_regex = r'^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+    return re.match(email_regex, email) is not None
+
+
+def main_overview() -> None:
+    """Creates main overview page on dashboard."""
+    st.title("Main Overview Page")
```

Comment:
Maybe the main title should include "Band Camp Tracker" somewhere and link to the website as well

Reply:
good idea, just added!